eos-image-boot-dm-setup

```diff
@@ -76,7 +76,7 @@ while read extent_i extent_size; do
 
   if [ ${i} -lt ${extent_i} ]; then
```

Comment:
I'm a bit suspicious of this comparison -- shouldn't this condition always be true, and wouldn't the mapped device get messed up if it's not?